tasks/ast_tools/src/output/mod.rs

```diff
@@ -22,6 +22,7 @@ pub enum Output {
     Rust { path: String, tokens: TokenStream },
     Javascript { path: String, code: String },
     Yaml { path: String, code: String },
+    Raw { path: String, code: String },
```

Comment:
nit: change the enum's documentation

Reply:
Good point. Thank you. I'll do that in a follow-up PR.